src/scripts/Controller.js

```diff
@@ -1200,19 +1234,52 @@ export default class SitemapController {
 		this._editSelector(selector, sitemap);
 	}
 
-	initConfirmActionPanel(action) {
+	initConfirmActionPanel(action, extraExtion = () => {}) {
```

Comment:
Here we should have all options likes sitemap._id or selector._id or childNum, is it copy mode or delete. Taht we needed to show a modal page.